.nais/vars/prod.yaml

```diff
@@ -0,0 +1,6 @@
+appName: sosialhjelp-soknad
+ingresses: null
+externalOutboundAccesses: null
```

Comment:
Tror kanskje du må legge til urlen for soknad-api i fss her. Se https://doc.nais.io/workloads/explanations/migrating-to-gcp/?h=pub.na#how-do-i-reach-an-application-found-on-premises-from-my-application-in-gcp

Reply:
Eller tenker du at denne skal gå på prod-gcp med en gang?

Reply:
Nei, dette var bare meg som hadde blingsa på proxy-oppsett, sorry.